scripts/openzfs-tracking.sh

```diff
@@ -84,22 +98,102 @@ while getopts 'hd:e:' OPTION; do
 done
 
 cat << EOF
+<!DOCTYPE html>
 <html>
 <head>
 <title>OpenZFS Tracking</title>
 <meta name="keyword" content="zfs, linux"/>
+<meta http-equiv="Content-type" content="text/html; charset=utf-8">
+<script		  src="https://code.jquery.com/jquery-1.12.4.min.js"
+			  integrity="sha256-ZosEbRLbNQzLpnKIkEdrPv7lOy9C27hHQ+Xp8a4MxAQ="
+			  crossorigin="anonymous"></script>
+<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.13/css/jquery.dataTables.min.css">
+<script type="text/javascript" language="javascript" src="https://cdn.datatables.net/1.10.13/js/jquery.dataTables.min.js"></script>
+
+<script type="text/javascript">
+\$(document).ready(function() {
+	\$('#maintable').DataTable( {
+		 "lengthMenu": [[50, 100, 500, -1], [50, 100, 500, "All"]],
+		 "order": [],
+		initComplete: function () {
+				var column = this.api().column(4);
+				var select = \$('<select><option value=""></option></select>')
+					.appendTo( \$(column.header()).empty() )
+					.on( 'change', function () {
+						var val = $.fn.dataTable.util.escapeRegex(
+							\$(this).val()
+						);
+
+						column
+							.search( val ? '^'+val+'$' : '', true, false )
+							.draw();
+					} );
+
+				column.data().unique().sort().each( function ( d, j ) {
```

Comment:
I think creating checkboxes is real. If everyone wants checkboxes, i can dig into it later.